bdk-ffi/src/descriptor.rs

```diff
@@ -355,6 +357,33 @@ impl Descriptor {
     pub fn desc_type(&self) -> DescriptorType {
         self.extended_descriptor.desc_type()
     }
+
+    pub fn derived_address(
```

Comment:
This method doesn't have a direct 1-to-1 match on the Rust side, but I was just thinking about how this is using a past tense/passive naming style vs the active `Descriptor::derive_address`. I think the convention is more on the active style, but this is a nit and not a blocker. Any opinion on this @reez?